pkg/protocol/verify.go

```diff
@@ -30,30 +25,30 @@ func stateVerificationHandler(ethStateContract *abi.State) packers.VerificationH
 
 // authV2CircuitStateVerification `authV2` circuit state verification
 func authV2CircuitStateVerification(contract *abi.State, pubsignals []string) error {
-	bytePubsig, err := json.Marshal(pubsignals)
-	if err != nil {
-		return err
-	}
-
-	authPubSignals := circuits.AuthV2PubSignals{}
-	err = authPubSignals.PubSignalsUnmarshal(bytePubsig)
-	if err != nil {
-		return err
-	}
-
-	globalState := authPubSignals.GISTRoot.BigInt()
-	globalStateInfo, err := contract.GetGISTRootInfo(&bind.CallOpts{}, globalState)
-	if err != nil {
-		return err
-	}
-
-	if globalState.Cmp(globalStateInfo.Root) != 0 {
-		return errors.Errorf("invalid global state info in the smart contract, expected root %s, got %s", globalState.String(), globalStateInfo.Root.String())
-	}
-
-	if (big.NewInt(0)).Cmp(globalStateInfo.ReplacedByRoot) != 0 && time.Since(time.Unix(globalStateInfo.ReplacedAtTimestamp.Int64(), 0)) > time.Minute*15 {
-		return errors.Errorf("global state is too old, replaced timestamp is %v", globalStateInfo.ReplacedAtTimestamp.Int64())
-	}
+	// bytePubsig, err := json.Marshal(pubsignals)
+	// if err != nil {
+	// 	return err
+	// }
+
+	// authPubSignals := circuits.AuthV2PubSignals{}
```

Comment:
do you need to comment everything???

Reply:
miss this. sorry. I did some tests yesterday.